support/client/lib/mil-sym/cws.js

```diff
@@ -5189,10 +5189,22 @@ define( function(){
                   "symbolType": "DefinedMissionGraphic",
                   "defaultControlPts": [ 0,10, 50,0, 50,0, 100,10 ],
                   "defaultDimensions": [100, 10]
+            },
+            "TACGRP.TSK.CATK": {
+                  "symbolID": "G*TPK-----****X",
+                  "tag": "COUNTERATTACK",
+                  "symbolType": "MultiSegmentArrow",
+                  "arrowLeafOffset": 40
+            },
+            "TACGRP.TSK.CATKF": {
+                  "symbolID": "G*TPKF----****X",
+                  "tag": "COUNTERATTACK BY FIRE",
```

Comment:
It is a little sensitive.  I did make some changes since you posted this comment that has helped to fix it.  However, if the user releases the mouse (or touch) with the last point just behind the arrow tip, it could result in a some weird behavior.  Editing the arrow afterwards will correct it, but programmatically, there has to be a way to filter out those errant points.

Reply:
Personally, even when I've been very careful, I haven't been able to create one that looks reasonable.  Maybe we should just comment this one out until we have time to look into filtering the points?